plugins/jobs/src/js/jobsDelete.js

```diff
@@ -0,0 +1,10 @@
+import StringUtil from "#SRC/js/utils/StringUtil";
+import UserActions from "#SRC/js/constants/UserActions";
+
+// TODO - use delete mediator https://jira.mesosphere.com/browse/DCOS-38492
+// add tests to it
+export default {
+  className: "text-danger",
+  label: StringUtil.capitalize(UserActions.DELETE)
+  // onItemSelect: this.props.handleDestroyButtonClick
```

Comment:
Why is this commented out?

Reply:
Sorry, I answered that on the previous time. this.props do not exist there, so this can't work. Notice that this file is not required anywhere.

Reply:
This should be made work by the ticket who is focused on the delete.

Reply:
ah okay, didn't see that it isn't required. Would you mind removing the line and adding a todo to add `onItemSelect` properly?

Reply:
I removed the line and enhanced the comment.